confluence/confluence-v1.asciidoc

```diff
@@ -0,0 +1,73 @@
+Confluence Rest Configuration
+----------------------------
+This documentation describs what data the confluence configuration retrives and the endpoints used.
+This is this list of data that rest connector using confluence configuration will crawl.
+
+* spaces 
+* pages (such as Wiki pages), 
+* blog posts,  
+* pages and blog posts comments (indexed as metadata) 
+
+Authentication
+~~~~~~~~~~~~~~~
+Support Basic Authentication with username (atlasian account) and API Token
+
+1. For more info see Documentation here - https://developer.atlassian.com/cloud/confluence/basic-auth-for-rest-apis/ 
+
+2. Check here to create new Token API https://id.atlassian.com/manage/api-tokens
+
+
+Supported Crawl
+~~~~~~~~~~~~~~~
+* Full Crawl supported
+* Re-Crawl supported - `rely on the strayContentDeletion`
+
+
+Rest Connector Variables used
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* ${LW_BATCH_SIZE} - This variable is used to set the `limit` query parameter which controls the number results are return in the response for both `blogpost` and `pages`
+
+* {LW_INDEX_START} - This variable  is used to set the `start` query parameter which use the travers the pagination. Confluence pagination is 0 based. This means the first page number is 0.
+
+* {LW_PARAM_KEY} - This variable is uesd to access a value from the response from `main request`to used in a `additional request`. Confluence use case this variable added to URL to proform a GET request, to get comments for blogpost and pages
+
+
+Endpoints to configure with Rest-Connector
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The table below consists of a list of endpoints and descriptions of what each endpoint retrieves.
+
+*Notes:*
+
+List of objects (spaces, pages, blogs, comments) are retrieved in batches (25 by default). Configure query limit to get more objects (the max size is 250)
+
+
+[options="header"]
+|=======================
+|Endpoint|Http Verb |Query Param |Description |Request type
+|/wiki/rest/api/content|GET    |type=blogpost&start=0&limit=1&expand=body.storage|get all blogpost from confluence|Data Request
+|/wiki/rest/api/content/${LW_PARAM_KEY}/child/comment|GET|expand=body.storage|This endpoint gets all blogpost comment. The value of `id` from the main request needs to be assign to the `${LW_PARAM_KEY}` variable so that the addtional feature can build insert the value to the get url.  |Child Endpoint
```

Comment:
Could change "feature can build insert the value to the get url" to
"feature can insert the value when building the GET url"